pkgs/by-name/bu/buttermanager/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, wrapGAppsHook
+, qt5
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "buttermanager";
+  version = "2.5.1";
+
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "egara";
+    repo = "buttermanager";
+    rev = version;
+    hash = "sha256-MLYJt7OMYlTFk8FCAlZJ1RGlFFXKfeAthWGp4JN+PfY=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    pyqt5
+    pyyaml
+    sip
+    tkinter
+    setuptools
+  ];
+
+  nativeBuildInputs = [ wrapGAppsHook qt5.wrapQtAppsHook ];
```

Comment:
As there are two wrapper hooks here, and, both are enabled, this is getting double-wrapped:

```
tree -a result/bin
result/bin
├── buttermanager
├── .buttermanager-wrapped
├── .buttermanager-wrapped_
└── ..buttermanager-wrapped-wrapped
```

The usual idiom to prevent this:

```nix
  dontWrapGApps = true;

  preFixup = ''
    qtWrapperArgs+=("''${gappsWrapperArgs[@]}")
  '';
  ```

Reply:
Correction, it's triple-wrapped due to the `buildPythonApplication`...

```nix
  dontWrapQtApps = true;
  dontWrapGApps = true;
  makeWrapperArgs = [ "\${qtWrapperArgs[@]}" "\${gappsWrapperArgs[@]}"];
```

Reply:
Thanks for catching! My first time packaging qt